test/unit/org/apache/cassandra/index/sai/memory/TrieMemtableIndexTestBase.java

```diff
@@ -250,6 +257,90 @@ public void indexIteratorTest()
         }
     }
 
+    @Test
+    public void updateCollectionTest()
```

Comment:
This is failing, I think it is due to some of the BM25 math, but I'm not certain. I'll review later.